forge/src/main/resources/META-INF/mods.toml

```diff
@@ -0,0 +1,18 @@
+modLoader = "javafml"
+loaderVersion = "[1,)"
```

Comment:
does this line say that it support forge loader since version 1? is this correct?

Reply:
yes, I temporarily left it at that because unlike fabric, the forge loader changes version with every Minecraft version and the goal is to support as far back as possible. But I will still need to test what is actually the minimum supported version

Reply:
I've been unable to test which version is exactly the minimum supported, but I don't think it's a huge issue to leave it as is. Even the official Forge MDK (template mod) sets the loader and forge version ranges to `[0,)` by default.

Reply:
ok, just leave it here and fix it when someone finds the incompatibility